### PR TITLE
feat: create introduction page and tutorial

### PR DESCRIPTION
**Pull request by fadnincx**
## Description

* Creates an introduction page
* Adds a small tutorial on how to get a default potos client

## Dependencies

n/a

### CODE REVIEW

File: docs/guide/iso-build/config.md [hunk 1]
```diff
@@ -18,26 +18,127 @@ The iso build is configured using [YAML](https://en.wikipedia.org/wiki/YAML) bas
...
+
+#### initial_user.password
+**Type**: *string*, **Default**: `admin` *hashed*
+
+The password in form of a hash. Create your own with `echo -n yourpasswordhere | mkpasswd --method=SHA-512 --stdin` .
+
+#### environment
+**Type**: *string*, **Default**: `production`
+
+Possible values are `production` and `develop`. The installation in `develop` mode is more verbose.
```
Comment (fadnincx) [inline docs/guide/iso-build/config.md:84]: Something like this, to have a red warning banner?
```suggestion
Possible values are `production` and `develop`. The installation in `develop` mode is more verbose.
::: danger DISCLAIMER
The develop mode potentially leaks confidential information to the end user!
:::
```

[the remark this answers is not in the document]
Comment (fadnincx) [reply docs/guide/iso-build/config.md:84]:
```suggestion
Possible values are `production` and `develop`. The installation in `develop` mode is more verbose.
::: danger DISCLAIMER
The develop mode potentially leaks confidential information to the end user!
:::
```

File: docs/guide/getting-started/README.md [hunk 2]
```diff
@@ -14,4 +16,45 @@ You can read everything about this "Specs" repository [here](/guide/specs-repo/o
...
+     3. Adjust in `config/config.yml` the following values to your new location
+        ```yaml
+        specs:
+          url: "https://github.com/projectpotos/"
+          repo: "ansible-specs-potos"
+          branch: "main"
+        ```
+     4. You may also want to adjust other values in `config/config.yml` such as `client_name`
+     5. While developing you may want to enable debugging with `environment: "develop"`, but his may leak confidential informations to the enduser, so ensure to change back before deploying an image.
+     5. Go back to build the image and test your
```
[the remark this answers is not in the document]
Comment (Daenou) [reply docs/guide/getting-started/README.md:56]: @fadnincx the sentence on L#56 appears to be incomplete

[the remark this answers is not in the document]
Comment (fadnincx) [reply docs/guide/getting-started/README.md:56]: Sorry missed that.
```suggestion
     5. Go back to build the image and test your changes. Your "Design - Build - Test - Learn" Cycle has just started :smiley: 
```

**Review by Daenou — APPROVED** [review]
LGTM :rocket:

**Pull request merged**